src/handlers.js

```diff
@@ -234,19 +234,22 @@ class Handlers {
 
   // `pointermove` and `mousemove` event handler
   handlePointerMove(event) {
```

Comment:
This function became a little bit messy.  IMU there are two major outcomes:

1. `getCoalescedEvents` does not return anything.  This way a single `event` must be "pushed" with `coalesced = false` flag.
2. `getCoalescedEvents` returns a few new events.  In this case the original event is discarded and the first few events are pushed with `coalesced = true` and the last one gets `false`.

Am I missing something here?  If this is so, please make it a little bit more readable.